gpkit/constraints/tight.py

```diff
@@ -3,6 +3,11 @@
 from ..nomials import PosynomialInequality, SignomialInequality
 
 
+class TightnessError(Exception):
+    "Raised when linked variables have differing values."
+    pass
```

Comment:
right. so I guess the high-level design question is whether

1. we write special error types that inherit from standard types for us to catch internally in special circumstances. Advantage: external code can more easily discriminate errors without knowing our special types, thanks to inheritance. Disadvantage: these might inadvertently be cause by other code catching standard types.

2. We write special error types to avoid detection by things catching errors. Advantage: easy to implement internally. Disadvantage: doesn't play nicely externally.